### PR TITLE
Feature/meeting created pop up #197

### PR DESCRIPTION
**Pull request by va-dym-ko-val**
image
image

### CODE REVIEW

File: frontend/src/app/shared/components/confirmation-window/confirmation-window.component.html [hunk 1]
```diff
@@ -1,11 +1,83 @@
...
+                <div class="left-participants">
+                    <h4>+{{ participants.length - maxVisibleMembers }}</h4>
+                </div>
+            </span>
+        </div>
+    </div>
+    <div *ngIf="location && link" class="created-meeting-options">
+        <div class="option">
+            <div class="option-name">Location</div>
+            <div class="option-value">{{ location }}</div>
```
Comment (Ymkavi) [inline frontend/src/app/shared/components/confirmation-window/confirmation-window.component.html]: This approach will cause the location type "Google Meet" to be written together.
image

File: frontend/src/app/shared/components/confirmation-window/confirmation-window.component.html [hunk 2]
```diff
@@ -1,11 +1,83 @@
...
+        <div class="option">
+            <div class="option-name">Link</div>
+            <div class="option-value-link">
+                <img src="assets/copy_icon.png" />
+                <div class="option-value link-value">{{ link }}</div>
+            </div>
+        </div>
+    </div>
+    <hr />
+    <div mat-dialog-actions [align]="'center'" class="mat-dialog-buttons">
```
Comment (Ymkavi) [inline frontend/src/app/shared/components/confirmation-window/confirmation-window.component.html]: "align" attribute for div is deprecated in Html5

File: frontend/src/app/shared/components/confirmation-window/confirmation-window.component.html [hunk 3]
```diff
@@ -1,11 +1,83 @@
...
+        </div>
+    </div>
+    <div *ngIf="participants" class="created-meeting-participants">
+        <div class="participants-label">
+            <div class="option-name">Participants: {{ participants.length }}</div>
+        </div>
+        <div class="participants-avatars">
+            <ng-container *ngFor="let participant of participants">
+                <ng-container *ngIf="participant.image; else noImageParticipant">
+                    <span class="avatar"><img src="{{ participant.image }}" /></span>
```
Comment (Ymkavi) [inline frontend/src/app/shared/components/confirmation-window/confirmation-window.component.html]: Please add the "alt" attribute to all your "img" tags

**Review by Ymkavi — CHANGES_REQUESTED** [review]

File: frontend/src/app/shared/components/confirmation-window/confirmation-window.component.html (hunk 1 above)
Comment (va-dym-ko-val) [reply frontend/src/app/shared/components/confirmation-window/confirmation-window.component.html]: image

**Review by Ymkavi — APPROVED** [review]

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html [hunk 4]
```diff
@@ -2,32 +2,46 @@
...
-                        <small *ngIf="meetingNameControl.errors?.['minlength']">Length must be more than 2
-                            characters.</small>
-                        <small *ngIf="meetingNameControl.errors?.['maxlength']">Length must be less than 50
-                            characters.</small>
+                        <small *ngIf="meetingNameControl.errors?.['minlength']"
+                            >Length must be more than 2 characters.</small
+                        >
+                        <small *ngIf="meetingNameControl.errors?.['maxlength']"
+                            >Length must be less than 50 characters.</small
+                        >
```
Comment (tatianahutii) [inline frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html]:
```suggestion
                        <small *ngIf="meetingNameControl.errors?.['minlength']">
                            Length must be more than 2 characters.
                        </small>
                        <small *ngIf="meetingNameControl.errors?.['maxlength']">
                            Length must be less than 50 characters.
                        </small>
```

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html [hunk 5]
```diff
@@ -36,17 +50,26 @@
...
-                                    {{unit}}
+                                    {{ unit }}
                                 </mat-option>
                             </mat-select>
                         </div>
                         <div *ngIf="customTimeControl.invalid && customTimeControl.touched">
-                            <small *ngIf="customTimeShown && customTimeControl.errors?.['required']">This field is required.</small>
+                            <small *ngIf="customTimeShown && customTimeControl.errors?.['required']"
+                                >This field is required.</small
+                            >
```
Comment (tatianahutii) [inline frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.html]: see the previous comment

File: frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts [hunk 6]
```diff
@@ -189,11 +210,42 @@ export class NewMeetingComponent extends BaseComponent implements OnInit {
...
+            titleImagePath: this.bookedIconPath,
+            date: '22 Aug 2022',
+            time: this.createdMeeting.startTime,
+            duration: this.duration,
+            meetingName: this.createdMeeting.name,
+            participants: getMembersForBookedWindow(),
+            location: this.createdMeeting.location,
+            link: this.createdMeeting.meetingLink,
+        });
+    }
```
Comment (tatianahutii) [inline frontend/src/app/modules/new-meeting/new-meeting/new-meeting.component.ts]: Please rework this as we discussed in our call

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/shared/components/booking-window/booking-window.component.ts [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+        this.title = data.title;
+        this.message = data.message;
+        this.titleImagePath = data.titleImagePath;
+        this.buttonsOptions = data.buttonsOptions;
+        this.dateTime = data.dateTime;
+        this.duration = data.duration;
+        this.meetingName = data.meetingName;
+        this.participants = data.participants;
+        this.location = data.location;
+        this.link = data.link;
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/booking-window/booking-window.component.ts]: why in the constructor and why do we need separate fields?

File: frontend/src/app/shared/components/confirmation-window/confirmation-window.component.sass [hunk 8]
```diff
@@ -1,12 +1,55 @@
...
+                padding-left: 30px
+    .message
+        font-family: 'Inter'
+        font-style: normal
+        font-weight: 400
+        font-size: 16px
+        padding-bottom: 20px
+        text-align: center
+    .mat-dialog-buttons
+        padding-top: 20px
```
Comment (tatianahutii) [inline frontend/src/app/shared/components/confirmation-window/confirmation-window.component.sass]: Are you sure we still need this?

**Review by tatianahutii — CHANGES_REQUESTED** [review]
Use icons instead of png

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**